api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py

```diff
@@ -186,3 +187,28 @@ def create_venues_across_cities() -> None:
                             seconds=random.randint(1, 59),
                         ),
                     )
+
+
+def create_offers_for_each_subcategory() -> None:
+    for subcategory in subcategories_v2.ALL_SUBCATEGORIES:
+        if subcategory.id in subcategories_v2.EVENT_SUBCATEGORIES:
+            offers_factories.EventStockFactory.create_batch(
+                size=10,
+                offer__product=None,
+                offer__subcategoryId=subcategory.id,
+                quantity=random.randint(10, 100),
+                beginningDatetime=datetime.datetime.utcnow()
+                + datetime.timedelta(
+                    days=random.randint(30, 59),
+                    hours=random.randint(1, 23),
+                    minutes=random.randint(1, 59),
+                    seconds=random.randint(1, 59),
+                ),
+            )
+        else:
+            offers_factories.StockFactory.create_batch(
+                size=10,
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Attention, les offres non-évènement n'ont qu'un seul stock. C'est la quantité du stock qui change

cf. le commentaire de Matthieu sur [cette PR](https://github.com/pass-culture/pass-culture-main/pull/8963)

Tu peux t'inspirer des autres `Offer.create_batch`

Reply:
dans l'autre PR je fixait l'offre , mais ici je ne fixe pas l'offre donc une nouvelle offre sera crée pour chaque stock

Reply:
Aah yes bien vu :+1: